bigbluebutton-html5/client/main.jsx

```diff
@@ -13,4 +13,6 @@ Meteor.startup(() => {
   window.addEventListener('error', (e) => {
     log('error', e);
   });
+
+  document.title = Meteor.settings.public.app.clientTitle;
```

Comment:
Would probably make more sense to set the document title to `meetingTitle - clientTitle`

Reply:
@oswaldoacauan see #5634